layouts/_partials/blocks/premium-newsletter.html

```diff
@@ -0,0 +1,138 @@
+<section class="@container mb-6 w-full bg-g-9 pt-12">
+  {{ $maxWidth := .Param "max-width" }}
+  {{ $maxWidthClass := cond (eq $maxWidth "prose") "max-w-prose" (print "max-w-[" $maxWidth "]") }}
```

Comment:
Does this work with Tailwind? This looks like the sort of code that might confuse its class name detector: https://tailwindcss.com/docs/detecting-classes-in-source-files#dynamic-class-names It might be simpler to just toggle between using `max-w-prose` and `max-w-(--spl-max)` and then add `style="--spl-max: {{ $maxWidth | safeCSS }};"` to the HTML.